### DISCUSSION TITLE
read mtext string contains Chinese character

### DISCUSSION THREAD

**Discussion by leateng** [mozman/ezdxf#859] [Q&A]
I open a DXF file which contains some Chinese character, 
when I print e.text(e is MText obj),  I got "\\M+1974C\\M+18CC0\\M+18CF6\\M+18E69"
what's this, and how could I convert this str to a human-readable utf-8 string?

**Comment by mozman** [comment]
Sorry, I have no idea how this encoding works.

**Comment by varunkaarthik** [comment]
It appears to be returning some sort of vector graphics data. You could search for the Chinese character directly in the dxf file (opening with a text editor) and see what it looks like- might help with solving your problem!

**Comment by firai** [comment]
I believe this is a multibyte string (MBCS), hence the `M+` marker. You probably need to know or guess at the underlying code page in order to convert it to Unicode.
